components/tools/ToolsDashboard.tsx

```diff
@@ -48,14 +44,6 @@ export default function ToolsDashboard() {
     };
   });
 
-  // sets the preloader on the page for 1 second
-  useEffect(() => {
-    setLoading(true);
-    setTimeout(() => {
-      setLoading(false);
-    }, 1000);
-  }, []);
-
```

Comment:
I have removed this timeout. and the loading state. Not sure why it was there before. Let me know if anything breaks because of this